packages/xo-server/src/models/user.mjs

```diff
@@ -36,9 +36,13 @@ export class Users extends Collection {
   }
 
   async update(properties) {
-    const { email } = properties
+    const { id, email } = properties
 
-    if (await this.exists({ email })) {
+    const currentUser = await this.first({ id })
```

Comment:
I think you don't need to fetch the current user, since you already have the id of the user